example/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.userleap.destination.example"
+        minSdk 26
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        // The Segment write key is configured with local.properties
+        Properties properties = new Properties()
+        properties.load(project.rootProject.file("local.properties").newDataInputStream())
+        buildConfigField("String", "SEGMENT_KEY", properties.getProperty("segment_key", ""))
```

Comment:
we should use `SEGMENT_WRITE_KEY` for this one because there are 2 keys in the segment dashboard

Reply:
👍 makes sense to make it less confusing